index.html

```diff
@@ -255,21 +255,16 @@
 
   <pre class="example js">
     // Gather server timing for a particular resource
```

Comment:
Would we have a stronger example if we changed it to "iterate over all resource timing entries, identify slow server processing times"? That is, change to .getEntryByType("resource") and forEach over that?